### DISCUSSION TITLE
How to configure router so that it automatically set routing based on directory just like nuxtjs?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuejs/router#1986] [Help and Questions] [answered]
I was wondering how to set up automatic routing so that it avoids the hassle of modifying the `router.ts` every time we add pages. It can also avoid conflicts especially if multiple people working on a project using version control like git. Thanks :)

**Comment by posva** [comment, marked as answer]
https://github.com/posva/unplugin-vue-router
